nmpolicy/internal/lexer/lexer.go

```diff
@@ -146,7 +148,7 @@ func (l *lexer) lexIdentity() (*Token, error) {
 
 		if l.isEOF() || l.isSpace() {
 			return token, nil
-		} else if l.isDot() || l.isEqual() || l.isVerticalTwoDots() {
+		} else if l.isDot() || l.isEqual() || l.isVerticalTwoDots() || l.isPlus() {
```

Comment:
Same should be added for `lexNumber` isn't it?
EDIT: since currently we support only merging of captures, we actually don't need it here atm.

Reply:
From token perspective if we find the plus rune the identity has finish.

Reply:
Fixed